windows/src/engine/kmcomapi/keymanapi_TLB.pas

```diff
@@ -1592,7 +1604,7 @@ interface
 // *********************************************************************//
   IKeymanKeyboardFile2Disp = dispinterface
     ['{EDE4326B-51F4-42D5-8251-B20B71993EC8}']
-    function Install2(Force: WordBool; InstallDefaultLanguage: WordBool): IKeymanKeyboardInstalled; dispid 403;
+    function Install2(Force: WordBool): IKeymanKeyboardInstalled; dispid 1610809344;
```

Comment:
wow, that's quite a diff from `403`

Reply:
The dispids are automatically generated; perhaps I should tidy them up...